examples/seismic/viscoacoustic/operators.py

```diff
@@ -0,0 +1,208 @@
+import sympy as sp
+import numpy as np
+
+from devito import (Eq, Operator, VectorTimeFunction, TimeFunction, NODE,
+                    div, grad, diag)
+from examples.seismic import PointSource, Receiver
+
+def viscoacoustic_blanch_symes(v, r, p, vp, irho, rho, t_s, tt, s, damp, **kwargs):
+    """
+    Stencil created from from Blanch and Symes (1995) / Dutta and Schuster (2014) 
+    viscoacoustic wave equation.
+
+    Parameters
+    ----------
+    v : VectorTimeFunction
+        The computed particle velocity.
+    r : TimeFunction
+        Memory variable.    
+    p : TimeFunction
+        The computed solution.
+    vp : Function or float
+        The time-constant velocity. 
+    irho : Function
+        The time-constant inverse density.
+    rho : Function
+        The time-constant density.
+    t_s : Float
+        The relaxation parameter.
+    tt : Float
+        The relaxation parameter.
```

Comment:
Same description, what is the difference?

Reply:
Okay. We will fix it.